components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ChefServerDetailsComponent } from './chef-server-details.component';
+
+xdescribe('ChefServerDetailsComponent', () => {
+  let component: ChefServerDetailsComponent;
+  let fixture: ComponentFixture<ChefServerDetailsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ ChefServerDetailsComponent ],
+      schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ChefServerDetailsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Important to add some tests, please.

Reply:
I have added some test cases, please have a look.